src/environments/environment.ts

```diff
@@ -0,0 +1,11 @@
+export const environment = {
```

Comment:
please create at least 2 env files, for dev and prod env.
They should be changed depending on build